### DISCUSSION TITLE
New episode titles are not available

### DISCUSSION THREAD

**Discussion by liferadioat** [skymethod/op3#33] [Q&A]
Hi,

we´re using op3 quite a while. Since january 2024 there a no new episodes listed at our podcast https://op3.dev/show/fbca525b-211a-5003-ab5c-131a7430b97b although we´re releasing a new episode every month. Is there any problem / bug with op3? How can we ensure that all downloads are tracked correctly? 

This is our feed: https://truecrime.at/feed/mp3/

Thanks for help.

**Comment by johnspurlock-skymethod** [comment]
Hi there - this can sometimes happen if multiple OP3 shows contain the same episodes, do you have any other feeds that contain the same episodes?  OP3 is conservative in this case and does not allocated to either show to avoid double counting.

I'll take a look on my side - don't worry as long as you have the prefix in your feed no download stats are lost, just temporarily unallocated.

**Reply by liferadioat** [reply]
Prior we had https://truecrime.at/feed/mobile/ which is redirecting to the https://truecrime.at/feed/mp3/ feed url. The only other feed is https://audio.liferadio.at/show/spurderverbrechen/feed/mp3/, which has the same first episode (trailer).

**Reply by liferadioat** [reply]
I´ve deleted the old feed with the trailer episode (https://audio.liferadio.at/show/spurderverbrechen/feed/mp3/) now, but I don´t think, that this was problem.

**Reply by johnspurlock-skymethod** [reply]
thanks, I'll take a look

**Comment by johnspurlock-skymethod** [comment]
So in this case, it was due to the fact that the enclosure urls had spaces in the names, and the way your host generates the enclosure urls inside the RSS feed in this case (they were not escaped).  Although that's not typical, it's something OP3 should be able to identify and workaround to improve its matching.

I deployed that workaround yesterday and reran stats going back to June 1st for your show (and others that had a similar problem). Rerunning stats like this takes a while, I'll try to rerun going back even further as time permits to pick up any other previously unallocated downloads that should be matched to your show.

Feel free to reach out via email or here asap if something doesn't look right, as rerunning after the fact is a fairly time- and resource-intensive process.

Hope that helps!

**Reply by liferadioat** [reply]
Thanks a lot - if your test succeeds the stats should be regenerated from the beginning of this year if it´s possible.

**Comment by princeofsky** [comment]
Hi,
Not sure if I should open a new discussion as I have the same problem: latest episode doesn't show on op3.
This is my feed: https://www.partirunjour.fr/-podcast/partirunjour.xml
Is there anything wrong?

**Reply by johnspurlock-skymethod** [reply]
I don't see an OP3 show for https://www.partirunjour.fr/-podcast/partirunjour.xml, but here's what I see:

1. OP3 show [d9d9e406d9e34dc99c37a2716d4250f6](https://op3.dev/show/d9d9e406d9e34dc99c37a2716d4250f6) added on 2024-11-03
Partir Un Jour
https://partirunjour.fr/-podcast/partirunjour.xml

2. OP3 show [ac1cda5f345d4ff09ce729ffa6af68b2](https://op3.dev/show/ac1cda5f345d4ff09ce729ffa6af68b2) added on 2025-10-11
Partir un jour | Note au voyageur
https://www.noteauvoyageur.eu/category/conseils/partir-un-jour/feed/

Unfortunately those two feeds reference the same enclosure urls. When a download is observed, if there is more than one possible OP3 show associated with the enclosure url, OP3 gives _neither_ show the credit for the download - not both, which would be double counting and throw off aggregate stats, and who can tell which feed was used anyway.

So OP3 has observed downloads since you added the prefix, but starting on 2025-10-11 they wouldn't have been associated with any OP3 shows. What I can do is delete the 2nd OP3 show (`2.` above) so that downloads will start being allocated to show `1.` starting with tonight's run going forward. Let me know asap if you'd rather keep `2.` as the winner.

If you'd like, email me (address is at bottom of the [op3.dev homepage](https://op3.dev)) and I can provide an export of any unallocated downloads OP3 has observed (based on your enclosure urls) since 2025-10-11 in spreadsheet form, but they won't appear on the public stats pages.

**Reply by princeofsky** [reply]
Yes, please delete the second OP3 show.
And nevermind the unallocated downloads since last october, we hadn't published any episode during this period.

Thanks for your help!

**Reply by Marzal** [reply]
Would using `https://op3.dev/e,pg=xxxx` prevent this?

I have a similar case with lots of PrestoCast's podcasts [1] and a general feed [2] (with all the episodes of the platform).

[1] https://podcastindex.org/podcast/7777377 (example)

[2] https://podcastindex.org/podcast/7750789
[2] https://podcastindex.org/podcast/7774582

**Reply by johnspurlock-skymethod** [reply]
@Marzal using the `pg` param is not necessary but never hurts, as long as it's correct (I've seen garbage). Also make sure the feed is public first - that is, make sure the feed is in the podcast index first, that's the main issue with prestocast feeds. The `pg` param is mostly used to find new feeds, not for deduplication.

Neither of those two aggregate feeds you listed in [2] have associated OP3 shows, so no unallocated downloads there.

**Reply by Marzal** [reply]
OK, thanks. 
PrestoCast it's trying to auto submit the shows to the PI with podping, but it's not currently working (Dave is aware).
So right now only the podcast with tech savvy podcasters are on PodcastIndex 😅
